roles/kubernetes-apps/ingress_controller/cert_manager/templates/00-namespace.yml.j2

```diff
@@ -5,3 +5,4 @@ metadata:
   name: {{ cert_manager_namespace }}
   labels:
     name: {{ cert_manager_namespace }}
+    certmanager.k8s.io/disable-validation: "true"
```

Comment:
What does this do since it's introduced ?

Reply:
I just copy and paste from upstream changes, by git diff v0.4.1..v0.5.0 ;-)

Reply:
but what does it do and why was it added?

Reply:
For the new `Resource Validation Webhook`, but we currently skip this in kubespray integration:

> To enable the component when using Helm, you must first ensure the namespace that you deploy cert-manager into has the label certmanager.k8s.io/disable-validation: "true".

See https://cert-manager.readthedocs.io/en/master/admin/resource-validation-webhook.html